tests/unit/action_recognition/test_action_recognition_notebooks.py

```diff
@@ -39,8 +44,8 @@ def test_01_notebook_run(action_recognition_notebooks):
         OUTPUT_NOTEBOOK,
         parameters=dict(
             PM_VERSION=pm.__version__,
-            MODEL_INPUT_SIZE=4,
-            EPOCHS=4,
+            MODEL_INPUT_SIZE=8,
+            EPOCHS=2,
```

Comment:
Not sure how slow action recognition training/inference exactly is, hence: are these notebook tests fast enough?

Reply:
yeah they aren't too slow at 2 and 8 respectively